packages/client/src/main.ts

```diff
@@ -1,2 +1,14 @@
-console.log('Client main.ts');
-export {}
\ No newline at end of file
+export interface Thing {
```

Comment:
I know it is garbage. I thought maybe tsc needed more substance to work properly before I realized it was VS Code hiding my dist folders.